docs/app/Examples/elements/Container/Types/ContainerTextExample.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+
+import React, { Component } from 'react'
+import { Container, Message } from 'stardust'
+
+export default class ContainerContainerExample extends Component {
+  render() {
+    return (
+      <div>
+        <Message className='info'>
+          <p>A text container is a simpler markup alternative to using a grid with a single column, and is designed to have a reasonable maximum width for displaying flowing text</p>
+        </Message>
```

Comment:
Since example file text appears on the site below the example, we want to keep it as minimal as possible.  This `<Message />` should be moved up to the ContainerTypesExamples.js by adding a closing tag to the `< ComponentExample />` for this example file:

```jsx
// ContainerTypesExamples.js ~ ln 14

<ComponentExample
   title='Text Container'
   description='A container can reduce its maximum width to more naturally accomodate a single column of text'
   examplePath='elements/Container/Types/ContainerTextExample'
 >
  <Message>
    // put the message here
    // it will be displayed above this example, but not in the example code
  </Message>
</ComponentExample>
```